tap_s3_csv/sync.py

```diff
@@ -61,13 +61,15 @@ def sync_table_file(config, s3_path, table_spec, stream):
         LOGGER.warning('"%s" without extension will not be synced.',s3_path)
         s3.skipped_files_count = s3.skipped_files_count + 1
         return 0
-    if extension == "zip":
-        return sync_compressed_file(config, s3_path, table_spec, stream)
-    if extension in ["csv", "gz", "jsonl", "txt"]:
-        return handle_file(config, s3_path, table_spec, stream, extension)
-
-    LOGGER.warning('"%s" having the ".%s" extension will not be synced.',s3_path,extension)
-    s3.skipped_files_count = s3.skipped_files_count + 1
+    try:
```

Comment:
@dbshah1212 Can you please provide a scenario in the comment about the UnicodeDecodeError and JSONDecodeError exceptions

Reply:
Added in except clause.